WORKSPACE

```diff
@@ -1,6 +1,6 @@
 git_repository(
     name = "io_bazel_rules_go",
-    commit = "d0142854a22a0dd98306280e897e64086289a0de",
```

Comment:
I'm not terribly concerned about this yet considering we aren't using bazel build artificats much. We should probably do the pinning in our repo eventually.